src/utils/i18n/i18n.ts

```diff
@@ -26,6 +34,20 @@ export const resolveFallbackLanguage = (primaryLanguage: string): string => {
   }
 };
 
+/**
+ * Each customer may own its own variation of the translations.
+ * Resolves the lang of the customer variation, based on the lang and the customer.
+ *
+ * XXX To define a customer variation language, you must create it on Locize manually
+ *  @example fr-x-customer1
+ *  @example en-x-customer1
+ *
+ * @param lang
+ */
+export const resolveCustomerVariationLang = (lang: string): string => {
```

Comment:
Not sure I understood this, is `fr-x-customer1` a namespace we should create on Locize to be able to use the customer variation languages?

Reply:
It's not a Locize namespace, but a Locize language. 

It's explained at https://nrn-v2-mst-aptd-at-lcz-sty-c1.vercel.app/fr/examples/built-in-features/static-i18n